### DISCUSSION TITLE
error Error: ER_ACCESS_DENIED_ERROR: Access denied for user 'root'@'localhost' (using password: NO)

### DISCUSSION THREAD

**Discussion by Msquittto** [opentiny/tiny-engine#363] [Q&A]
本地启动data-service时， npm run dev出现报错error Error: ER_ACCESS_DENIED_ERROR: Access denied for user 'root'@'localhost' (using password: NO)

**Comment by lu-yg** [comment]
你配置文件写mysql密码了吗
